signals.go

```diff
@@ -99,8 +99,9 @@ func (h *signalHandler) forward(process *libcontainer.Process, tty *tty, detach
 				}
 			}
 		default:
-			logrus.Debugf("sending signal to process %s", s)
-			if err := unix.Kill(pid1, s.(unix.Signal)); err != nil {
+			us := s.(unix.Signal)
+			logrus.Debugf("forwarding signal %d (%s) to %d", int(us), unix.SignalName(us), pid1)
```

Comment:
One more thought for consideration (I wanted to suggest so earlier, but now also see similar code is used above in this function).

We could add `pid`, `signal`, and `signal_name` (e.g.) as fields; something like;

```go
logrus.WithFields(logrus.Fields{
    "pid":    pid1,
    "signal": us,
    "signal_name": unix.SignalName(us),
}).Debug("forwarding signal")
```

Reply:
I thought about that, too, but at this time I'd rather not add more structured logging, as I guess at some point we'll need to replace logrus...

Reply:
Yeah, but it's still there, and it would be producing somewhat more consistent logs.

Anyway, not a blocker